<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReosTelemacEngineConfigurationDialog</class>
 <widget class="QDialog" name="ReosTelemacEngineConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TELEMAC Engine Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Additional pathes</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Configuration name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>CPU Usage Count</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="mCPUSpinBox">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="mTelemacConfigFileButton">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QToolButton" name="mTelemac2DPythonScriptButton">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TELEMAC 2D Python script</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Python path</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="mLineEditConfig">
       <property name="toolTip">
        <string>If void, the first configuration in the configuraton file will be considered</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Configuration file</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="mTelemac2DPythonScriptLineEdit"/>
     </item>
     <item row="6" column="1">
      <widget class="QTextEdit" name="mDependenciesPathTextEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="mTelemacConfigFileLineEdit">
       <property name="toolTip">
        <string>Configuration file can have a name as systel.***.cfg</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="mPythonPathLineEdit"/>
     </item>
     <item row="5" column="2">
      <widget class="QToolButton" name="mPythonPathButton">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="mDefaultButton">
       <property name="text">
        <string>Reset to Default</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReosTelemacEngineConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReosTelemacEngineConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
